src/shammodels/ramses/include/shammodels/ramses/SolverConfig.hpp

```diff
@@ -78,7 +78,14 @@ namespace shammodels::basegodunov {
             return false;
         }
     };
+    /**
+     * @brief Npscal_gas is the number of gas passive scalars
+     */
+    struct PassiveScalarGasConfig {
+        u32 npscal_gas = 0;
 
+        inline bool is_gas_passive_scalar_on() { return npscal_gas == 0; }
+    };
```

Comment:
```suggestion
    };

```
An extra line jump to avoid hurting people's eyes